newrelic-python.ini

```diff
@@ -45,47 +48,27 @@ app_name = your_app_name
 # NEW_RELIC_MONITOR_MODE environment variable.
 monitor_mode = true
 
-# When enabled, the agent's security module is not loaded
-# this would mean changes on soft reloads won't work
-security.force_complete_disable = false
-# When enabled, the agent collects security data about your
-# application and reports this data to the New Relic.
-security.enable = true
+# To completely disable security set flag to false If the flag is
+# set to false, the security module is not loaded. This property
+# is read only once at application start.
+security.agent.enabled = agent_enabled_flag
+
+# Indicates if attack detection security module is to be enabled
+security.enabled = security_enabled_flag
+
 # security module provides two modes IAST or RASP
 # RASP stands for Runtime Application Self Protection
 # while IAST for Interactive Application Security Testing
 # Default mode is RASP
 security.mode = RASP
 
 # web-protect agent endpoint connection URLs
-security.validator_service_endpoint_url = ws://your_validator_system_ip:54321
-security.resource_service_endpoint_url = http://your_validator_system_ip:54322
-
-# log level
-security.log_level = DEBUG
-
-# security file settings
-security.sec_home_path = /tmp
+security.validator_service_url = ws://your_validator_system_ip:54321
```

Comment:
placeholder value for validator endpoint is "your_validator_system_ip" instead of "ws://your_validator_system_ip:54321". Please refer other config files

Reply:
Update